plugins/woocommerce-beta-tester/src/live-branches/components/BranchList.tsx

```diff
@@ -153,7 +150,9 @@ export const BranchList = ( { branches }: { branches: Branch[] } ) => {
 					) }
 					{ ! activeBranch && <WooCommerceVersionInfo /> }
 				</CardBody>
-				<CardFooter></CardFooter>
+				<CardFooter>
+					<Fragment />
+				</CardFooter>
```

Comment:
Added a Fragment because children is a required prop in the `CardFooter` component.